### PR TITLE
Project view

### LINKED ISSUE #262: Update project view to look awesome

### PR DESCRIPTION
**Pull request by Christiaanvdm**
This closes #262 
NewGIF

### CODE REVIEW

File: django_project/core/settings/contrib.py [hunk 1]
```diff
@@ -6,10 +6,10 @@
 import locale
 import os
 from .base import *  # noqa
-from .secret import (
-    COMMENTS_DISQUS_API_PUBLIC_KEY,
-    COMMENTS_DISQUS_API_SECRET_KEY,
-    COMMENTS_DISQUS_SHORTNAME)
+# from .secret import (
```
Comment (Christiaanvdm) [inline django_project/core/settings/contrib.py]: This caused me errors... Should I undo these changes before the push?

File: django_project/mezzanine_people/migrations/0001_initial.py [hunk 2]
```diff
@@ -1,11 +1,9 @@
 # -*- coding: utf-8 -*-
-from south.utils import datetime_utils as datetime
-from south.db import db
-from south.v2 import SchemaMigration
-from django.db import models
+from django.db import models, migrations
```
Comment (Christiaanvdm) [inline django_project/mezzanine_people/migrations/0001_initial.py]: I could also not import south even after trying various ways to install it... replacing it with the modern django libraries worked a charm though!

File: django_project/kartoza_theme/templatetags/mezzanine_people/people_tags.py [hunk 3]
```diff
@@ -1,58 +1,58 @@
...
-    try:
-        tag_name, arg = token.contents.split(None, 1)
-    except ValueError:
-        raise template.TemplateSyntaxError, "%s tag requires arguments" % token.contents.split()[0]
-    m = re.search(r'(.*?) as (\w+)', arg)
-    if not m:
-        raise template.TemplateSyntaxError, "%s tag had invalid arguments" % tag_name
-    format_string, var_name = m.groups()
-    return RandomPeople(format_string[0], var_name)
+# from django.db.models import Count
```
Comment (Christiaanvdm) [inline django_project/kartoza_theme/templatetags/mezzanine_people/people_tags.py]: These tags weren't being used anywhere and were stopping me from compiling.

File: deployment/docker/REQUIREMENTS.txt [hunk 4]
```diff
@@ -1,15 +1,15 @@
-cartridge
+cartridge==0.12
```
Comment (Christiaanvdm) [inline deployment/docker/REQUIREMENTS.txt]: I updated versions for the modules to stop all the dependancy errors.

**Review by dimasciput — APPROVED** [review]
LGTM
